azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -29,13 +31,16 @@ def to_pandas_timedelta(raw_value: Union[int, float, str]) -> "pandas.Timedelta"
             return pd.to_timedelta(formatted_value)
 
 
-def dataframe_from_result_table(table: "Union[KustoResultTable, KustoStreamingResultTable]", nullable_bools: bool = False) -> "pandas.DataFrame":
+def dataframe_from_result_table(
+    table: "Union[KustoResultTable, KustoStreamingResultTable]", nullable_bools: bool = False, column_name_totype_dict: dict = {}, type_totype_dict: dict = {}
```

Comment:
Technically the converters can do a lot more than just change the type since it's a lambda - they can filter them, format them, etc. So I think the general name is good (+ shorter)